Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner.js';
+import CardContent from './CardContent.js';
+
+
+const CardContainer = () => <div class="cardContainer"><div class="border"><CardBanner /><CardContent /></div></div>;
```

Comment:
This works, but it would be much easier to read this if it were spaced out on multiple lines!

```
const CardContainer = () => {
  return (
    <div class="cardContainer">
      <div class="border">
        <CardBanner />
        <CardContent />
      </div>
    </div>
  )
}
```